client/src/components/Form.js

```diff
@@ -0,0 +1,88 @@
+import styled from "@emotion/styled";
+import React from "react";
+import { useForm } from "react-hook-form";
+
+export default function Form() {
+  const { register, handleSubmit, errors } = useForm();
+  const onSubmit = (data) => console.log(data);
+  console.log(errors);
+
+  const Form = styled.div`
+    display: flex;
+    flex-grow: 1;
+  `;
+
+  return (
+    <Form>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <input
+          type="text"
+          placeholder="Straße und Hausnr."
+          name="Straße"
+          ref={register({
+            required: true,
+            min: 2,
+            maxLength: 15,
+            pattern: /^\S+@\S+$/i,
+          })}
+        />
+        <input
+          type="number"
+          placeholder="PLZ"
+          name="PLZ"
+          ref={register({ required: true, min: 5, maxLength: 6 })}
```

Comment:
I guess we can use either min and max or minLenght and maxLenght, right?

Reply:
Didn't knew that before, thank you!